automation/Commands/utils/webdriver_extensions.py

```diff
@@ -9,7 +9,7 @@
 import random
 import time
 
-import domain_utils as du
+from ....automation.utilities import domain_utils as du
```

Comment:
Nit: You should just be able to do `from ...utilities import domain_utils as du` since this file and the utilities file are both under `automation`.